### ISSUE TITLE
README 👀

### ISSUE THREAD

**Issue by Markel** [MarkelFe/metacritic-crawler#19]
I'm going to edit the README, as some things are needed to be added and others modified. If you have any extra ideas to include leave a comment (in the same way, if you think that any of them shouldn't be include say it):

1. Improve the _Description_ section explaining what every script does. 
2. Probably I will finish #13 so the installation process is done faster and simpler. Also probably win32api isn't necessary as it is being discussed at #14 
3. #16 included parameters, which even if in the commit says that are hardcoded (corrected on a comment) they can be modified thought flags, a section naming them would be helpful. Also the flag -s is interesting to mention.
4. There is an extra dash on the Meta section
5. With all the changes above the Usage section will probably need to be modified too.
6. _How to interpret the data_ section, although I will probably create a separate page (or wiki) for that.

**Comment by cdrani** [comment]
1) If everything thing is moved to the readme then you wouldn't need all the extra comments at the top of games.py and analyze.py.
2) Are you on windows, and if so does the script make use of win32 at all? I know for a fact that win32 would not work on a unix os like macos or linux.
3) The parameters are not hardcoded though. They have default values if not provided. These are spider parameters which are added through the cli:

`scrapy runspider -a delay=5 -a start_page=145 -o gm.jl`

User parameters use `-a` flag and `-s` flag is for setting/overriding some defaults. One that should be provided should be the `CLOSESPIDER_PAGECOUNT` so that the spider closes gracefully after it has finished running:

`scrapy runspider -s CLOSESPIDER_PAGECOUNT=2 -a delay=5 -a start_page=145 -o gm.jl`

**Comment by Markel** [comment]
1. I don't know, sometimes I prefer to have the comments also in the files for various reasons: First, they can be updated much easier with makes them almost never to be outdated. Secondly (and finally), I really prefer to read the top line of a file than having to read a separate file while I'm developing (and commands, etc are easier to access). Anyway I'll probably summarize them.

3. Yeah, I correct my error (late), interesting -s flag I'm adding it.

If you have any other ideas comment them ^^

**Comment by cdrani** [comment]
Fair enough. All I'm saying is that most people's will read the readme first, hence the name, prior to diving into the code itself, if ever. Something like the text in the games.py:

> This file creates an spider that gets the links of all the games
For that it uses the best games page of Metacritic which has all the games (around 15K on December 2018)
It usually takes between 40 and 80 minutes to make a complete run
To run it use the command 'scrapy runspider games.py -o gm.jl'
Remember that you MUST comply with the point 11.13 of the terms of use in Metacritic (https://www.cbsinteractive.com/legal/cbsi/terms-of-use), you need the permission of Cbs Interactive to run any kind of scraping (this is a scraping tool if you didn't know it btw)
You bear all responsibility about the use of this program 

should just be in the readme. `You bear all responsibility about the use of this program ` is also in  the licence I believe.

**Comment by Markel** [comment]
Refer to the `#19-branch` if you want to contribute to this 😁
